src/EFCore.PG/Storage/Internal/Mapping/NpgsqlArrayTypeMapping.cs

```diff
@@ -150,14 +165,16 @@ static TElem[] Snapshot(TElem[] source, ValueComparer<TElem> elementComparer)
             }
         }
 
-        class SingleDimComparerWithIEquatable<TElem> : ValueComparer<TElem[]>
-            where TElem : IEquatable<TElem>
+        /// <inheritdoc />
+        class SingleDimComparerWithIEquatable<TElem> : ValueComparer<TElem[]> where TElem : IEquatable<TElem>
```

Comment:
Leave the constraint on the next line.

Reply:
Any reason in particular? Even as a one-liner, it's only at column 110, which is shorter than other lines in the file.

Reply:
In the project we prefer to put constraints on the next lines everywhere. The reason is simple: this improves readability since a constraint is placed at fixed offset. Even if you haven't finished to read the type definition line, you find that it's a generic method or type. Also it keeps the same style when you have multiple type constraints.

Reply:
Reverted.